### PR TITLE
center: rv v1.6.1 release for hydrophone and camera

### PR DESCRIPTION
**Pull request by trekhopton**
This new version supports i2s for the MEMs microphone.

### CODE REVIEW

**Review by ao-david — APPROVED** [review]
Nice!

**Review by scruzin — APPROVED** [review]

**Review by scruzin — COMMENTED** [review]
This PR includes changes that don't relate to oceancenter.

File: cmd/oceancenter/dl/hydrophone/pkg/v1.0.1/pkg.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "device": "hydrophone",
+  "description": "Install rv in /opt/ausocean/bin.",   
+  "version": "v1.0.0",
+  "components": [
+    {
+    "name": "rv",
```
Comment (scruzin) [inline cmd/oceancenter/dl/hydrophone/pkg/v1.0.1/pkg.json:7]: Make _rv/v1.6.1_ a shared component.

**Review by scruzin — CHANGES_REQUESTED** [review]

File: cmd/oceancenter/dl/hydrophone/pkg/v1.0.1/pkg.json (hunk 1 above)
Comment (trekhopton) [reply cmd/oceancenter/dl/hydrophone/pkg/v1.0.1/pkg.json:7]: Isn't that what I've done here by adding 
"shared": true,
?

**Comment by trekhopton** [comment]
> This PR includes changes that don't relate to oceancenter.

What changes? every file that has been changed is in oceancenter.

**Review by scruzin — APPROVED** [review]

**Pull request merged**